packages/core/src/renderTexture/RenderTexture.ts

```diff
@@ -151,8 +167,10 @@ export class RenderTexture extends Texture
      * @param {number} [height]
      * @param {PIXI.SCALE_MODES} [scaleMode=PIXI.settings.SCALE_MODE]
      * @param {number} [resolution=1]
+     * @param {PIXI.MSAA_QUALITY} [multisample=PIXI.MSAA_QUALITY.NONE]
      */
-    static create(width: number, height: number, scaleMode?: SCALE_MODES, resolution?: number): RenderTexture;
+    static create(width: number, height: number, scaleMode?: SCALE_MODES, resolution?: number,
+        multisample?: MSAA_QUALITY): RenderTexture;
```

Comment:
Same, please don't add more to deprecated methods. The approach here is object-based param.